frontend/packages/eslint-plugin-console/lib/config/prettier.js

```diff
@@ -0,0 +1,49 @@
+const merge = require('merge');
+
+module.exports = {
+  extends: ['prettier', 'prettier/react'],
+
+  plugins: ['prettier'],
+
+  rules: {
+    'prettier/prettier': [
+      'error',
+      {
+        singleQuote: true,
+        trailingComma: 'all',
+        bracketSpacing: true,
+        jsxBracketSameLine: false,
+        arrowParens: 'always',
+        printWidth: 120,
+        semi: true,
+        useTabs: false,
+      },
+      {
+        usePrettierrc: false,
+      },
+    ],
+  },
+
+  overrides: [
+    {
+      files: ['*.ts', '*.tsx'],
+      rules: merge(require('./rules/prettier-typescript'), {
+        'prettier/prettier': [
+          'error',
+          {
+            parser: 'typescript',
+            singleQuote: true,
+            trailingComma: 'all',
```

Comment:
Personally, I would prefer using "es5" here as "all" puts commas also to function arguments e.g.:
```
...
).then(
  (m) => m.BaremetalHostsPage,
),
```

Reply:
I'm indifferent as the tool takes care of it for me. My only reservation is that with any function that accepts optional parameters, a change to add a new parameter will affect 2 lines instead of 1 which is the argument for always including trailing commas.
Whatever the general consensus is.
What else does the `es5` setting value affect?

Reply:
I prefer having trailing commas always because it makes it easy to add and remove arguments without worrying about the last comma. It's also like Go, which we use on the backend and in our operator.

But I don't care all that much as long we are enforcing a consistent style :) I'm good starting with any reasonable set of rules as the tool formats the code for us. We can always tweak later if there's something we don't like.

Reply:
Ok, good reasoning. Let's stick with 'all'. Thanks!